.github/workflows/log_management.yml

```diff
@@ -21,7 +24,10 @@ jobs:
           retain-run-count: 0
           dry-run: false
       - name: Workflow Housekeeper - workflows in default branch
+
         uses: JosiahSiegel/workflow-housekeeper@731cc20bb613208b34efb6ac74aab4ba147abb50
+        ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/workflow-housekeeper
```

Comment:
corp email please

Reply:
I have updated the email address to reflect the Aquia corporate email.